test/e2e/node/node_problem_detector.go

```diff
@@ -384,9 +384,13 @@ func getNpdPodStat(ctx context.Context, f *framework.Framework, nodeName string)
 		if !strings.HasPrefix(pod.PodRef.Name, "node-problem-detector") {
 			continue
 		}
-		cpuUsage = float64(*pod.CPU.UsageNanoCores) * 1e-9
-		rss = float64(*pod.Memory.RSSBytes) / 1024 / 1024
-		workingSet = float64(*pod.Memory.WorkingSetBytes) / 1024 / 1024
+		if pod.CPU != nil {
```

Comment:
```suggestion
		```suggestion
		if pod.CPU.UsageNanoCores != nil {
```
Technically you still could get a panic if UsageNanoCores was undefined.
```

Reply:
you are right. i gotta check nil on both as well.